tools/fp20compiler/main.cpp

```diff
@@ -34,27 +34,80 @@ nvparse_errors errors;
 int line_number;
 char * myin = 0;
 
+static char* find_at_line_start(char* haystack, char* cursor, const char* needle) {
+    cursor = strstr(cursor, needle);
+    while(cursor != NULL) {
```

Comment:
No space between `while` and `(`, not sure how I feel about that.

Reply:
I've changed this to make it more consistent with the rest of main.cpp and most of the nvidia code.